openedx/features/ucsd_features/models.py

```diff
@@ -0,0 +1,64 @@
+from django.conf import settings
+from django.db import models
+
+
+# Backwards compatible settings.AUTH_USER_MODEL
+USER_MODEL = getattr(settings, 'AUTH_USER_MODEL', 'auth.User')
+
+
+class AdditionalRegistrationFields(models.Model):
+    """
+    This model contains two extra fields that will be saved when a user registers.
+    The form that wraps this model is in the forms.py file.
+    """
+    user = models.OneToOneField(USER_MODEL, null=True)
+
+    gender_nb = models.CharField(blank=False, max_length=32, verbose_name=b'Gender', choices=[
+        (b'male', b'Male'),
```

Comment:
Is there any specific reason for using bytes here?

Reply:
No idea.